### ISSUE TITLE
[Feb 2025] Accessibility Audit: Conduct tests page user flows

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#8519]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the Conduct tests page user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/queue`

Use cases to test:
* empty queue page
* multiple patients in queue
* adding test using patient search (inc search returned no results and multiple results)
* backdating a result
* multiple and single disease devices
* sensitive topics modal, where results are sent modal
* test timer
* success toast (stretch: try an error toast as well?)

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

### Additional context
See tickets from previous audits #5774, #6327

**Comment by emyl3** [comment]
Findings:
`/app/queue` - empty queue [dev.simplereport.gov-Conduct-test-empty-queue-2025-02-28.csv](https://github.com/user-attachments/files/19027975/dev.simplereport.gov-Conduct-test-empty-queue-2025-02-28.csv)

**Comment by emyl3** [comment]
Issues:
- https://github.com/CDCgov/prime-simplereport/issues/8549

**Comment by emyl3** [comment]
Ready for review
